pkgpanda/build/__init__.py

```diff
@@ -958,11 +958,12 @@ def cache_abs(filename):
             pkg_path = repository.package_path(pkg_id_str)
             pkg_tar = pkg_id_str + '.tar.xz'
             if not os.path.exists(package_store.get_package_cache_folder(requires_name) + '/' + pkg_tar):
-                raise BuildError("The build tarball {} refered to by the last_build file of the "
-                                 "dependency {} variant {} doesn't exist. Rebuild the dependency.".format(
-                                    pkg_tar,
-                                    requires_name,
-                                    requires_variant))
+                raise BuildError(
+                    "The build tarball {} refered to by the last_build file of the dependency {} "
+                    "variant {} doesn't exist. Rebuild the dependency.".format(
+                        pkg_tar,
+                        requires_name,
+                        requires_variant))
```

Comment:
I'm always in favor of pulling code "to the left" as much as possible, so I very much like this change.

But: was this change now enforced by one of the new checks? I would not support that, because both of these styles are PEP 8 compliant

```
foo = long_function_name(var_one, var_two,
                         var_three, var_four)
```

```
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```